main.go

```diff
@@ -32,6 +32,12 @@ import (
 	"github.com/peterhellberg/link"
 )
 
+// Struct to be used for milestone queries
+type milestone struct {
+	DueDate string
+	ID      string
```

Comment:
For easy of use we might also add a "Title" here. Even so we are using map[string/title]milestone{} it might be cleaner to iterate over the map and use milestone.Title.